tests/instruments/hp/test_hp8753e.py

```diff
@@ -0,0 +1,32 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2023 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import numpy as np
+
+from pymeasure.adapters import PrologixAdapter
+from pymeasure.instruments.hp import HP8753E
+
```

Comment:
Here an example, which should work:
```suggestion
from pymeasure.test import expected_protocol

def test_start_frequency_setter():
    with expected_protocol(
        HP8753E,
        [("STAR 1.2345e6 Hz", None)],
        name="test",  # in order to not trigger reading the id.
    ) as inst:
        inst.start_frequency = 123_456


def test_start_frequency_getter():
    with expected_protocol(
        HP8753E,
        [("STAR?", "123456")],
        name="test",  # in order to not trigger reading the id.
    ) as inst:
        assert inst.start_frequency == 123_456
```

Reply:
I used this method to get _some_ code coverage but I will have to work on improving it in the future. One of the biggest issues I have encountered is the instrument having an apparent non-deterministic delay on completing the same commands in the same sequence.